factory/installScripts/node/default.sh

```diff
@@ -23,7 +23,7 @@ ARCH= && dpkgArch="$(dpkg --print-architecture)" \
       141F07595B7B3FFE74309A937405533BE57C7D57 \
       74F12602B6F1C4E913FAA37AD3A89613643B6201 \
       DD792F5973C6DE52C432CBDAC77ABFA00DDBF2B7 \
-      61FC681DFB92A079F1685E77973F295594EC4689 \
```

Comment:
I suggest not to remove this key as it is listed in https://github.com/nodejs/node/blob/main/README.md#release-keys for previous releases. Removing it may prevent using `factory` to use an older version of Node.js